controllers/numaresourcesscheduler_controller.go

```diff
@@ -112,8 +112,26 @@ func (r *NUMAResourcesSchedulerReconciler) Reconcile(ctx context.Context, req ct
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *NUMAResourcesSchedulerReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// we want to initiate reconcile loop only on change under labels or spec of the object
```

Comment:
so, did you figure out why the `Own` did not work before? what was the root cause?

Reply:
Yes we do! I used the ```SetOwnerReference``` function instead of the ```SetControllerReference``` which doesn't trigger the controller's reconcile loop.

Thanks to @cynepco3hahue  for that.

Reply:
Great, TIL.